include/grpc/impl/grpc_types.h

```diff
@@ -467,6 +467,8 @@ typedef struct {
 /** If set, uses a local subchannel pool within the channel. Otherwise, uses the
  * global subchannel pool. */
 #define GRPC_ARG_USE_LOCAL_SUBCHANNEL_POOL "grpc.use_local_subchannel_pool"
+/** If set, privacy. */
```

Comment:
Let's omit from the public API and just include it in a header we can visibility protect?

Reply:
removed arg.